### ISSUE TITLE
Rename bioimage.yaml in "required file" text to "rdf.yaml"

### ISSUE THREAD

**Issue by jmetz** [bioimage-io/bioimageio-uploader#32]
As decision on this renaming was to stick with original for now.

**Comment by FynnBe** [comment]
Wasn't the consensus to switch to `bioimageio.yaml`??

**Comment by jmetz** [comment]
😅 @oeway asked me to switch it back... I'll let you two discuss 🫣

**Comment by oeway** [comment]
> Wasn't the consensus to switch to `bioimageio.yaml`??

No, if I remember correctly the concensus was to not touch it, see the last item in the meeting minutes here: https://github.com/bioimage-io/bioimage.io/issues/28#issuecomment-1802276877

**Comment by FynnBe** [comment]
ah, thanks, sorry for the confusion!

**Comment by mese79** [comment]
Anyway, I've made a PR:
https://github.com/bioimage-io/bioimageio-uploader/pull/93

**Comment by FynnBe** [comment]
ah, I remember: I did change how the collection handles it, so that on bioimage.io it's all `rdf.yaml`...
I suppose the PR would still be good to avoid confusion
